layers/auto-completion/packages.el

```diff
@@ -263,18 +264,6 @@
                                                                eshell-mode-hook)))
     :config
     (progn
```

Comment:
Actually this is not cosmetic, if the argument for `:config` or any other keyword is multi-line then `progn` must be used. It makes the code more readable and easier to work with using paredit like commands or navigation. I'll add this to the conventions.